sycl/test/inline-asm/asm_multiple_instructions.cpp

```diff
@@ -1,3 +1,4 @@
+// UNSUPPORTED: cuda
```

Comment:
no impact on CUDA RT. Please do not apply this change

Reply:
@vladimirlaz , the CUDA LIT change has been removed.